src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/WeekNum.txt

```diff
@@ -11525,76 +11525,76 @@
 
```

Comment:
I'm all for having many tests, but in this case we're going overboard. Do we really need 4000-ish cases to cover this function? As long as we cover the interesting cases (early dates, beginning / end of year, end of February / beginning of March, different startofweek arguments), we should have complete coverage of this function. We should also have test for earlier dates (i.e., before 1000) and later dates (after 9000) which I couldn't find (maybe there are, but it's hard to find in the middle of so many unnecessary tests.

Reply:
This file was previously in the "NotReadyYet" folder. We can leave it as is for now since we need to unblock other teams and work on cutting some fat from this txt file in an after-match issue. @MikeStall thoughts?

Reply:
I typically don't like to leave things to do later - they have the tendency of getting pushed out by some other priorities. Do the other teams really need this now, or can they wait another 1-2 days?